src/config.js

```diff
@@ -14,4 +14,6 @@ export default {
   enableReduxLogging: isDevelopment && !!global.btoa,
   enableSentry: !isDevelopment,
   sentryKey: 'ADD-DSN-HERE',
+  activePingInterval: 50 * 1000, // seconds between 2 presence fetch api call
```

Comment:
@Sam1301 changed it to milliseconds.

@borisyankov In order to maintain self status as active we need to ping focus in fixed time interval and in return we are getting presence. https://github.com/zulip/zulip/blob/c4eeb1335361fed7223f8a1d69b83c6e806e971b/static/js/activity.js#L11